Sources/PaymentButtons/Assets.xcassets/Redesign Logos/paypal_monogram_credit.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Do we need a separate `Redesign Logos` and `Logos` directory within Assets? I think we can just replace the ones in `Logos` without needing 2 separate directories

Reply:
That's true. I wasn't sure if we needed to keep the old logos for anything so I separated them so that it would be easier to go through the `ImageAsset` logic. I'll go ahead and remove the old logos

Reply:
Gotcha - yeh I think we can 🧨 nuke the old ones!